src/libs/ReportUtils.ts

```diff
@@ -773,11 +774,12 @@ function isAnnounceRoom(report: OnyxEntry<Report>): boolean {
     return getChatType(report) === CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE;
 }
 
+const chatTypes = [CONST.REPORT.CHAT_TYPE.POLICY_ADMINS, CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE, CONST.REPORT.CHAT_TYPE.DOMAIN_ALL];
```

Comment:
move this to CONST as `DEFAULT_POLICY_ROOM_CHAT_TYPES`

Reply:
Only reason to not move this to `CONST` was that we will now have same string in two places and if we update it in one place, we will have to do it in other place as well.

For eg, below is what the `CONST` will look like if I add that:

```js
const CONST = {
   // This doesn't work !!
    DEFAULT_POLICY_ROOM_CHAT_TYPES: [CONST.REPORT.CHAT_TYPE.POLICY_ADMINS, CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE, CONST.REPORT.CHAT_TYPE.DOMAIN_ALL],
   ...otherProperties,
   REPORT: {
        CHAT_TYPE: {
            POLICY_ANNOUNCE: 'policyAnnounce',
            POLICY_ADMINS: 'policyAdmins',
            DOMAIN_ALL: 'domainAll',
            POLICY_ROOM: 'policyRoom',
            POLICY_EXPENSE_CHAT: 'policyExpenseChat',
        },
   }
```

Since the above doesn't work, we will have to add magic strings in `DEFAULT_POLICY_ROOM_CHAT_TYPES`:

```js
const CONST = {
    DEFAULT_POLICY_ROOM_CHAT_TYPES: ['policyAdmins', 'policyAnnounce', 'domainAll'],
   ...otherProperties,
   REPORT: {
        CHAT_TYPE: {
            POLICY_ANNOUNCE: 'policyAnnounce',
            POLICY_ADMINS: 'policyAdmins',
            DOMAIN_ALL: 'domainAll',
            POLICY_ROOM: 'policyRoom',
            POLICY_EXPENSE_CHAT: 'policyExpenseChat',
        },
   }
```

Now, we have same string in two places and each time if we update any, we'll have to update other one as well. Thoughts?

Reply:
you could also just add separate const for the duplicated strings above the `const CONST = {...}`, right? Or does that break the const-ness in TS?